src/view/dataElements/xdmObject.jsx

```diff
@@ -248,24 +248,36 @@ const XdmExtensionView = () => {
           imsAccess: initInfo.tokens.imsAccess
         })
           .then(response => {
-            let sandbox = { name: null };
+            let sandbox;
             // if sandboxes are returned from the api call
             if (response.sandboxes && response.sandboxes.length) {
               setSandboxes(response);
-              // default to the first item in the list
-              sandbox = response.sandboxes.find(
-                _sandbox => _sandbox.isDefault === true
-              );
```

Comment:
This is where the issue was. We wouldn't find a sandbox with `isDefault` set to `true`, leaving `sandbox` undefined.